### ISSUE TITLE
Cannot reference library layer

### ISSUE THREAD

**Issue by Vittly** [bem/bem-sdk#262]
My bemrc:

```
{
    // ...
    sets: {
        desktop: '@foo-lib/common common'
    }
}
```

not working because [only set reference](https://github.com/bem/bem-sdk/blob/master/packages/config/index.js#L338) is supported

**Comment by qfox** [comment]
@Vittly Ouh. As far as I remember we did it on purpose because we decided that user should reveive a complete set of levels which represents a complete code.
But when you asking for a layer you'll get just a part of code and in most cases you'll broke the order between sets.

The main idea was to publish only complete sets or layers instead of partial.

@tadatuta Correct me please if I'm wrong.

**Comment by Vittly** [comment]
I agree referencing specific layer is kind of hack. Lets close it and fix `README.md`:

> // Will use `touch-phone` set from bem-components and few local levels
> "touch-phone": "@bem-components/touch-phone common touch touch-phone",

**Comment by veged** [comment]
Реализовано в 11bffb7 (PR #398).

После аудита оказалось, что `@lib/layer` reference (например, `'@foo-lib/common common'`) **уже корректно** парсится текущим `resolveSets` как `[{library:'foo-lib', layer:'common'}, {layer:'common'}]`. Добавил регрессионный тест на канонический кейс из issue + JSDoc-документацию семантики, чтобы не потерять.

Заодно улучшил error message на неоднозначной форме `set@lib/layer` — теперь явно объясняется, что эта комбинация не поддерживается и какие альтернативы есть (`@lib/layer` для library layer, `set@lib` для library set).

**Disambiguation** `@lib/X` (set vs layer): полноценная развязка по содержимому библиотеки в `resolveSets` невозможна — функция чистая, не знает о `libs`. Правило явное и предсказуемое:
- `@lib/X` всегда **layer**
- `X@lib` всегда **set**

Если нужен set `common` из библиотеки — используется форма `common@foo-lib`.
